opendbc/safety/tests/test_hyundai.py

```diff
@@ -173,7 +173,7 @@ class TestHyundaiLegacySafetyEV(TestHyundaiSafety):
   def setUp(self):
     self.packer = CANPackerPanda("hyundai_kia_generic")
     self.safety = libsafety_py.libsafety
-    self.safety.set_safety_hooks(CarParams.SafetyModel.hyundaiLegacy, 1)
+    self.safety.set_safety_hooks(CarParams.SafetyModel.hyundaiLegacy, HyundaiSafetyFlags.EV_GAS)
```

Comment:
separate fix, another PR

Reply:
Reverted.

Reply:
I mean you can open a PR now

Reply:
Gotcha, opened: https://github.com/commaai/opendbc/pull/1969